spec/ordy/orm/active_record/orderable_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require './spec/support/orm/active_record/active_record_test_db'
+
+describe Ordy::Orm::ActiveRecord::Orderable do
```

Comment:
We also need the specs that check that current AR scope is not being overridden...for example, calling `#order_by` after some filtering:

```rb
User.where(status: :active).order_by(:last_name, :desc)
```

Also, we need to test default ordering